diff --git a/game.py b/game.py
index e7d9a4d..2821fe0 100644
--- a/game.py
+++ b/game.py
@@ -402,7 +402,7 @@ class Game():
         self._svg_width = self._dot_size * 2
         self._svg_height = self._dot_size * 2
         self._stroke = '#101010'
-        self._fill = '#404040'
+        self._fill = '#00FF11'
         return svg_str_to_pixbuf(
             self._header() + \
             self._turtle() + \
